### PR TITLE
added id for sorting

### PR DESCRIPTION
**Pull request by BrianZito**
**Description:**
Fixed bug that caused 500 error when attempting to sort subawards by id

**Technical details:**
subawards.py was not using the dictionary lookup to translate 'id' to 'subaward_id' in order to match with ORM object.

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] API documentation updated
3. [x] Necessary PR reviewers:
    - [x] Backend
4. [N/A ] Matview impact assessment completed
5. [N/A ] Frontend impact assessment completed
6. [N/A ] Data validation completed
7. [N/A ] Appropriate Operations ticket(s) created
8. [x] Jira Ticket [DEV-2798](https://federal-spending-transparency.atlassian.net/browse/DEV-2798):
    - [x] Link to this Pull-Request
    - [ ] Performance evaluation of affected (API | Script | Download)
    - [ ] Before / After data comparison

**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

File: usaspending_api/awards/tests/unit/test_subawards.py [hunk 1]
```diff
@@ -88,7 +96,7 @@ def test_specific_award(mock_matviews_qs):
         " chips cosby echo etsy forage future helvetica kale occupy salvia sartorial semiotics skateboard squid"
         " williamsburg yr. 8-bit banh beer before they sold out craft ethnic fingerstache fixie irony jean shorts"
         " life organic park photo booth retro salvia tattooed trade vhs williamsburg.",
-    "action_date": "2017-09-30",
+    "action_date": "2017-09-29",
```
Comment (tony-sappe) [inline usaspending_api/awards/tests/unit/test_subawards.py]: Curious why this date was changed

File: usaspending_api/awards/tests/unit/test_subawards.py [hunk 2]
```diff
@@ -49,13 +49,22 @@ def test_all_subawards(mock_matviews_qs):
     subawards_logic = svs._business_logic(test_params)
     assert [] == subawards_logic
 
+    sort("id",[strip_award_id(subaward_3), strip_award_id(subaward_1), strip_award_id(subaward_2)])
+    sort("amount", [strip_award_id(subaward_3), strip_award_id(subaward_2), strip_award_id(subaward_1)])
+    sort("action_date", [strip_award_id(subaward_2), strip_award_id(subaward_1), strip_award_id(subaward_3)])
+    sort("recipient_name", [strip_award_id(subaward_2), strip_award_id(subaward_3), strip_award_id(subaward_1)])
+
+def sort(sort,response):
```
Comment (tony-sappe) [inline usaspending_api/awards/tests/unit/test_subawards.py]: Good idea to create a function to test numerous sort values. Please rename `sort()` to be more descriptive that it is testing sort behavior

Comment (tony-sappe) [reply usaspending_api/awards/tests/unit/test_subawards.py]: Also please change `response` to `expected_response`

File: usaspending_api/awards/tests/unit/test_subawards.py (hunk 1 above)
Comment (BrianZito) [reply usaspending_api/awards/tests/unit/test_subawards.py]: so it doesn't tie with the other date. Might be a good idea to test sorting the the case of ties, but I'm lazy.

**Pull request merged**